ValetTouchIDTest/ValetTouchIDTestViewController.swift

```diff
@@ -18,74 +18,69 @@
 //  limitations under the License.
 //
 
+import Valet
 import UIKit
 
 
 final class ValetTouchIDTestViewController : UIViewController
 {
     // MARK: Properties
-
+    
     @IBOutlet var textView : UITextView?
-    var singlePromptSecureEnclaveValet : VALSinglePromptSecureEnclaveValet
+    var singlePromptSecureEnclaveValet : SinglePromptSecureEnclaveValet
     let username = "CustomerPresentProof"
-
-
+    
     // MARK: Initializers
-
+    
     required init?(coder aDecoder: NSCoder)
     {
-        if let valet = VALSinglePromptSecureEnclaveValet(identifier: "UserPresence", accessControl: VALAccessControl.userPresence) {
-            self.singlePromptSecureEnclaveValet = valet
-        } else {
-            return nil;
-        }
-
+        singlePromptSecureEnclaveValet = SinglePromptSecureEnclaveValet.valet(with: Identifier(nonEmpty: "UserPresence")!, accessControl: .userPresence)
+        
         super.init(coder: aDecoder)
     }
-
-
+    
     // MARK: Outlets
-
+    
```

Comment:
Do you mean to be adding this trailing whitespace?

Reply:
yeah. We're reverting back to Xcode defaults. Not entirely sure how we ended up missing this whitespace to begin with.